paymentsource/types.go

```diff
@@ -0,0 +1,6 @@
+package paymentsource
```

Comment:
rather than putting this in its own package, can we put it in `payment_source.go` and name it `PaymentSource{BitcoinReceiver,Card}`